test/unit/performance/index.spec.js

```diff
@@ -8,6 +8,14 @@ var perfModule = require('../../../lib/performance');
 
 describe('lib/performance', function () {
 
+  it('exposes a getEntries function', function () {
+    expect(perfModule.getEntries).to.be.a('function');
+  });
+
+  it('exposes a getEntriesByName function', function () {
```

Comment:
Should we have one for getEntriesByType too?

Reply:
Yes we should, I'll get that added to this PR.